src/protocols/rdp/rdp_settings.c

```diff
@@ -1265,6 +1277,9 @@ void guac_rdp_push_settings(guac_rdp_settings* guac_settings, freerdp* rdp) {
 #endif
 
     /* Device redirection */
+    if (guac_settings->timezone)
+        setenv("TZ", guac_settings->timezone, 1);
```

Comment:
A warning should be logged if this cannot be set, since an expectation of the behavior of the connection is going unfulfilled.

Reply:
It appears that, in order to do this, I would need to pass the entire `client` object through to this `guac_rdp_push_settings()` function?

Reply:
A pointer to it, yeah.